src/mvViewport_win32.cpp

```diff
@@ -146,30 +156,34 @@ mvHandleMsg(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam) noexcept
 				cheight = crect.bottom - crect.top;
 			}
 
-			viewport->actualWidth = awidth;
-			viewport->actualHeight = aheight;
-
-
-			if (viewport->decorated)
 			{
-				GContext->viewport->clientHeight = cheight;
-				GContext->viewport->clientWidth = cwidth;
-			}
-			else
-			{
-				GContext->viewport->clientHeight = cheight;
-				GContext->viewport->clientWidth = cwidth;
-			}
-
-			//GContext->viewport->resized = true;
-			mvOnResize();
-			GContext->viewport->resized = false;
-
-			if (mvToolManager::GetFontManager().isInvalid())
-			{
-				mvToolManager::GetFontManager().rebuildAtlas();
-				ImGui_ImplDX11_InvalidateDeviceObjects();
-				mvToolManager::GetFontManager().updateAtlas();
+				std::lock_guard<std::recursive_mutex> lk(GContext->mutex);
+
+					viewport->actualWidth = awidth;
+					viewport->actualHeight = aheight;
+
+
+				if (viewport->decorated)
+				{
+					GContext->viewport->clientHeight = cheight;
+					GContext->viewport->clientWidth = cwidth;
+				}
+				else
+				{
+					GContext->viewport->clientHeight = cheight + 39;
```

Comment:
Doesn't this break #1130 ? Not sure where these lines came from, I didn't have them in the original fix...

Reply:
Ooh. I believe so. Let me check (this was several fixes in a single commit).

Reply:
Have you had a chance to look at this? Any plans on how to handle it? Since this PR is closed and seems to have been included into 1.9, we probably need to open another PR to change the lines back, right?

I'm asking because I've started updating to 1.9 and would like to have a clean base upon which to cherry-pick my commits...